.devcontainer/Dockerfile

```diff
@@ -15,6 +15,7 @@ RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y \
     lcov \
     nano \
     pmccabe \
+    python-is-python3 \
```

Comment:
Can you remove? `cppcheck` HTML report generation uses `python2` so it needs `python` as `python2` kinda dumb I know... :P